src/Symfony/Component/DependencyInjection/EnvVarProcessor.php

```diff
@@ -206,6 +207,12 @@ public function getEnv(string $prefix, string $name, \Closure $getEnv): mixed
             return null;
         }
 
+        if ('shuffle' === $prefix) {
+            \is_array($env) ? shuffle($env) : throw new RuntimeException(sprintf('Env var "%s" cannot be shuffled because its value "%s" is not an array.', $name, $env));
```

Comment:
`$env` in the exception is not necessarily a string here, right?

Reply:
what do you suggest, var_export?